### DISCUSSION TITLE
Add something like music control

### DISCUSSION THREAD

**Discussion by Friendclubattle78** [maximmax42/Discord-CustomRP#120] [Ideas]
It would be good if it add feature such 24/7 like when you closed the application then the **Rich Presence** still running until you disconnect it then it will stop.

**Comment by maximmax42** [comment]
I'm not sure what you mean. If you mean to have presence running without the app window being active all the time, you can just close the window and it'll be hidden to tray. If you want it to run even when pc is off, then that's impossible with my app, it needs active connection with Discord.

**Reply by Friendclubattle78** [reply]
> I'm not sure what you mean. If you mean to have presence running without the app window being active all the time, you can just close the window and it'll be hidden to tray. If you want it to run even when pc is off, then that's impossible with my app, it needs active connection with Discord.

Oh so it impossible? ;-;

**Comment by Friendclubattle78** [comment]
I dont know if you can add this like when you listening some music in some web (ex: Spotify,Youtube,Deezer...) it will show up in this
@maximmax42 
![image](https://user-images.githubusercontent.com/86119688/168329324-7e049e79-d683-4cab-b2cd-bd35c3fb5257.png)

**Reply by maximmax42** [reply]
That's in presence? I don't think I can.
